sapl/materia/forms.py

```diff
@@ -521,6 +532,19 @@ class Meta:
             'data_fim_prazo': forms.DateInput(format='%d/%m/%Y'),
         }
 
+
+    def __init__(self, *args, **kwargs):
+        if 'data' in kwargs and 'instance' in kwargs:
+            data_dict = dict(kwargs['data'].items())
+            data_dict['unidade_tramitacao_local'] = kwargs['instance'].unidade_tramitacao_local_id
+            data_dict['data_tramitacao'] = kwargs['instance'].data_tramitacao
+            from django.http.request import QueryDict
+            qdict = QueryDict('', mutable=True)
+            qdict.update(data_dict)
+            kwargs.pop('data')
+            kwargs['data'] = qdict
```

Comment:
```suggestion
            kwargs['data'] = kwargs['data'].copy()
            kwargs['data']['unidade_tramitacao_local'] = kwargs['instance'].unidade_tramitacao_local_id
            kwargs['data']['data_tramitacao'] = kwargs['instance'].data_tramitacao
```
As linhas 538 a 545 podem ser substituidas pelas 3 linhas acima. O seu objetivo era editar um QueryDict imutável. Para isso, basta criar uma copia dele, que ele já se tornará mutável.